virttest/migration.py

```diff
@@ -297,7 +297,10 @@ def do_migration(self, vms, srcuri, desturi, migration_type,
                         elif func == virsh.migrate_postcopy:
                             func(vm.name, uri=srcuri, debug=True)
                         else:
-                            func(args['func_params'])
+                            if 'func_params' in args:
```

Comment:
Do you need check args['func_params'] = None situation? which may leads to unexpected result when call func(args['func_params'])

Reply:
Some function need a argument which is None, So just check if func_params are in args here.